sample.tf

```diff
@@ -10,7 +10,7 @@ terraform {
 
 provider "aws" {
   profile = "default"
-  region  = "us-west-2"
+  region  = "us-west-1"
 }
 
 resource "aws_instance" "app_server" {
```

Comment:
Issue - Ensure that your AWS application is not deployed within the default Virtual Private Cloud in order to follow security best practices 
 category - Infrastructure Security 
 severity - MEDIUM